tailwind/ui/__stories__/accordion.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import { Meta, StoryObj } from "@storybook/react"
+
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from "../accordion"
+
+const meta = {
+  title: "Molecules / Disclosure Content / Accordions",
+  component: Accordion,
+  decorators: [
+    (Story) => (
+      <div className="w-[300px]">
+        <Story />
+      </div>
+    ),
+  ],
+} satisfies Meta<typeof Accordion>
+
+export default meta
+
+export const Basic: StoryObj = {
```

Comment:
Found complication here.

Passing the type of `Accordion` (which is actually `Accordion.Root` from Radix) to the `meta` object ends up with the `args` option being given the type `never`, though it should be required in this setup.

Seems like the type of this component is pretty complex and `StoryObj` might not be acknowledging it and assuming it's not a component type.